src/components/Nav/RegistrationForm.jsx

```diff
@@ -33,7 +32,7 @@ export default function RegistrationForm() {
     password: "",
     primaryemail: "",
     iimageurl: "",
-    terms: ""
+    terms: "",
```

Comment:
interesting. last trailing comma in an obj is purely aesthetic choice, right? was it just prettify that did this or is there another reason why like edge cases or performance? Purely curious.

Reply:
This is another one I didn't touch, prettier again.